installing/installing_aws/ipi/installing-aws-outposts.adoc

```diff
@@ -1,13 +1,89 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="installing-aws-outposts"]
-= Installing a cluster with compute nodes on AWS Outposts
+= Extending an AWS VPC cluster into an AWS Outpost
```

Comment:
Consider attributizing "AWS" as per the common-attributes file. To avoid any renderings issues, you can move the attributes include as per the note in [this guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata).

Reply:
I'll leave this as is since it's preexisting content (this assembly was really just renamed rather than deleted, but the PR presents it as the old one was deleted and this was a new assembly).